contrib/cf/pcf-tile/Makefile

```diff
@@ -0,0 +1,38 @@
+# Checks for the existence of a docker client and prints a nice error message
+# if it isn't present
+.PHONY: check-docker
+check-docker:
+	@if [ -z $$(which docker) ]; then \
+		echo "Missing \`docker\` client which is required for development"; \
+		exit 2; \
+	fi
+
+.PHONY: pull-tile-generator
+pull-tile-generator: check-docker
+	docker pull cfplatformeng/tile-generator:v11.0.5
+
+SHELL = /bin/bash
```

Comment:
Can we move all variable definitions like this one up to the top of the Makefile, please?

Reply:
Thanks for the suggestion, will do soon.